modelci/finetuner/README.md

```diff
@@ -1,7 +1,84 @@
 # Finetuner
 
-## Fine tune for Computer Vision
-Tune ResNet18 on CIFAR10 for 25 epochs:
-```shell script
-python computer_vision_fune_tuning.py
+\[Experimental\] Finetuner is a tiny module of ModelCI for easy fine-tuning pre-trained model on a new dataset. 
+
+Finetuner **packs the training job** with specified Transfer Learning algorithm, dataset, and pre-trained model 
+written on various ML framework (e.g. PyTorch). Such training jobs are **managed and schedules** by a coordinator. 
+We are to support various training environment, such as local machine, private cluster, and public cloud platform. 
+
```

Comment:
Besides, we plan to adopt auto tuning tools such as NNI and autoKeras into the system to free users from heavy training hyper-parameter tunining tasks.